### ISSUE TITLE
NSButton: 0x7fe573a48080

### ISSUE THREAD

**Issue by omerkymcu** [kochlisGit/ProphitBet-Soccer-Bets-Predictor#83]
Hello, I have successfully completed the installation, but I'm getting the following error and encountering the issue shown in the screenshot. How can I fix this?

024-04-15 23:28:53.160 python[10142:131011] Warning: Expected min height of view: (<NSButton: 0x7fe573a48080>) to be less than or equal to 30 but got a height of 32.000000. This error will be logged once per view in violation.
![Uploading Ekran Resmi 2024-04-15 23.41.34.png…]()

**Comment by kochlisGit** [comment]
Hm, that's strange. Have you tried other league? Also, do you have internet connection?

**Comment by omerkymcu** [comment]
You are right, I accessed the data when I connected via VPN. Thanks for the tip.
